### DISCUSSION TITLE
How to set up a decision making agent using autogen ?

### DISCUSSION THREAD

**Discussion by kuppan4109** [microsoft/autogen#2515] [Q&A]
### Discussed in https://github.com/microsoft/autogen/discussions/2477

<div type='discussions-op-text'>

<sup>Originally posted by **kuppan4109** April 22, 2024</sup>
Context : To create a single chat interface for multiple agents 
Agents to be incorporated : HR Assistant and Admin Assistant

Goal: I'm trying to create a multiagent framework where the chat manager should be decision making whether which chat assistant to pick 

Example:

User Ask : What are my payroll benefits?

Chat Manager : User is asking something regarding to employee pay benefits , I should ask HR Assistant about it 

Note: the process shouldn't happen sequential , chat manager should intelligently identify which agent to pick based on the context of user ask without any manual intervention

@sonichi @qingyun-wu Please have a look and address this 

</div>

**Comment by ekzhu** [comment]
Have you tried GroupChat? https://microsoft.github.io/autogen/docs/tutorial/conversation-patterns#group-chat

In the future can you use Discussions instead of Issues for Q&A?
